service/routes.py

```diff
@@ -5,11 +5,93 @@
 """
 
 from flask import jsonify, request, abort, make_response
+
+# from flask_restx import Api, Resource
+from flask_restx import fields, reqparse
 from service.common import status  # HTTP Status Codes
 from service.models import Wishlist, WishlistItem
 
 # Import Flask application
-from . import app
+from . import app, api
+
+# MODELS
+
+create_item_model = api.model(
+    "WishlistItem",
+    {
+        "wishlist_id": fields.Integer(
+            required=True, description="The Unique ID of the wishlist for the item"
+        ),
+        "product_name": fields.String(
+            required=True, description="The name of the product in the wishlist"
+        ),
+        "product_id": fields.Integer(
+            required=True, description="The ID of the product in the wishlist"
+        ),
+        "item_quantity": fields.Integer(
+            required=True, description="The quantity of the product in the wishlist"
+        ),
+    },
+)
+
+item_model = api.inherit(
+    "ItemModel",
+    create_item_model,
+    {
+        "id": fields.Integer(
+            readOnly=True, description="The unique id assigned internally by service"
+        ),
+    },
+)
+
+
+create_wishlist_model = api.model(
+    "Wishlist",
+    {
+        "name": fields.String(required=True, description="The Name of the wishlist"),
+        "owner_id": fields.Integer(
+            required=True, description="The owner id of the wishlist"
+        ),
+    },
+)
```

Comment:
I think we now also have a "is Public:" field as well

Reply:
Hi @studentgrad0419 , I think the 'public function' is already in my code, 
<img width="440" alt="image" src="https://github.com/CSCI-GA-2820-FA23-003/wishlists/assets/112911684/365f3e10-df0f-436b-ab1d-656c795bc836">

Reply:
<img width="674" alt="image" src="https://github.com/CSCI-GA-2820-FA23-003/wishlists/assets/112911684/218483b2-b8da-44e7-b672-8478c77d39b2">
I think it's ready to merge now.